examples/plot_05_train_test_split.py

```diff
@@ -138,39 +143,48 @@
 X_train, X_test, y_train, y_test = skore.train_test_split(
     X=X, y=y, test_size=0.2, random_state=0
 )
-print(X_train, "\n", y_train, "\n", X_test, "\n", y_test)
+X_train_explicit = X_train.copy()
 
 # %%
 # Moreover, when passing ``X`` and ``y`` explicitly, the ``X``'s are always returned
 # before the ``y``'s, even when they are inverted:
 
 # %%
-arr = np.arange(10).reshape((5, 2))
+arr = X.copy()
 arr_train, arr_test, X_train, X_test, y_train, y_test = skore.train_test_split(
     arr, y=y, X=X, test_size=0.2, random_state=0
 )
-print(X_train, "\n", y_train, "\n", X_test, "\n", y_test)
+X_train_explicit_inverted = X_train.copy()
+
+print("When expliciting, with the small typo, are the `X_train`'s still the same?")
+print(np.allclose(X_train_explicit, X_train_explicit_inverted))
+
 
 # %%
 # Automatic diagnostics: raising methodological warnings
 # ------------------------------------------------------
 #
+# In this section, we show how skore can provide methodological checks.
+#
+# Class-imbalance
+# ^^^^^^^^^^^^^^^
+#
 # In machine learning, class-imbalance (the classes in a dataset are not equally
 # represented) requires a specific modelling.
 # For example, in a dataset with 95% majority class (class ``1``) and 5% minority class
 # (class ``0``), a dummy model that always predicts class ``1`` will have a 95%
 # accuracy, while it would be useless for identifying examples of class ``0``.
 # Hence, it is important to detect when we have class-imbalance.
 #
-# Suppose that we have imbalanced data :
+# Suppose that we have imbalanced data:
 
 # %%
-X = [[1]] * 4
-y = [0, 1, 1, 1]
+X = np.arange(10_000).reshape((5_000, 2))
+y = [0] * 4_000 + [1] * 1_000
 
 # %%
-# In that case, :func:`skore.train_test_split` raises a warning telling the user that
-# there is class imbalance:
+# In that case, :func:`skore.train_test_split` raises a ``HighClassImbalanceWarning``
+# warning telling the user that there is class imbalance:
```

Comment:
```suggestion
# telling the user that there is class imbalance:
```

Reply:
not sure which is better, i'll let you decide

Reply:
same for all other instances of "... raise a `MyWarning` warning"

Reply:
I mean that this passage isn't the only place where the docs say "warning" twice like that, so if you change it here then you should change it in the other places too

Reply:
Going with "warning"